src/super_gradients/training/pretrained_models.py

```diff
@@ -24,6 +24,7 @@
               "yolo_v5m_coco": "https://deci-pretrained-models.s3.amazonaws.com/YOLOv5+m/ckpt_best.pth",
               "yolo_v5l_coco": "https://deci-pretrained-models.s3.amazonaws.com/YOLOv5+l/ckpt_best.pth",
               "yolo_v5n_coco": "https://deci-pretrained-models.s3.amazonaws.com/YOLOv5+n/ckpt_best.pth",
+              "ssd_lite_mobilenet_v2": "https://deci-pretrained-models.s3.amazonaws.com/ssd_lite_mobilenet_v2/ckpt_best.pth",
```

Comment:
Can you please add the usual transfer learning and mAP verification tests to tests/integration_tests/pretrained_models_test.py?

Reply:
Added in b1e22b9, 00814be, 471143a